OWNERS_ALIASES

```diff
@@ -161,3 +161,15 @@ aliases:
     - pawanpinjarkar
     - rwsu
     - zaneb
+  imagebased-reviewers:
+    - eranco74
+    - javipolo
+    - mresvanis
+    - patrickdillon
+    - tsorya
+    - zaneb
```

Comment:
I don't need more review requests from the bot :stuck_out_tongue:

Reply:
I marked you and Patrick only as approvers :+1: